version 2

DannyParker0001 opened the pull request:
REFACTORED code to allow for RAW ARRAYS and IMPROVED PERFORMANCE.

DannyParker0001 committed:
version 2

BowDown097 reviewed (commented):
Is there a possibility of squeezing some more comedy out of this? It is very important that we ensure our codebase is as comedic as possible before deployment for the best possible user experience :smiley:

DannyParker0001 committed:
Improved code claridy and increased comedic value

I believe the new when the based system is the future of programming, its significantly easier to follow. Will be seeing much use in my future projects!
Using the when the system I was able to make more references to comedy and in the when the infrastructure I was able to reference Allah basmala brothers.

DannyParker0001 commented:
I believe this latest commit reaches our comedic expectations however I'm going to keep this PR as a draft for now, I believe I could further optimize this function. It needs some investigating.

DannyParker0001 committed:
Performance improvements

Use a SIMD instruction to avoid a unaligned memcpy. (and memcpy call entirely)
Increase the the usage
add some assumes to help the compiler

BowDown097 commented:
wtf does this do
